docs/client-intro.rst

```diff
@@ -86,5 +86,5 @@ one before you can interact with tables or data.
 
 Head next to learn about the :doc:`instance-api`.
 
-.. _Instance Admin: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/tree/master/bigtable-protos/src/main/proto/google/bigtable/admin/instance/v1
-.. _Table Admin: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/tree/master/bigtable-protos/src/main/proto/google/bigtable/admin/table/v1
+.. _Instance Admin: https://github.com/googleapis/java-bigtable/tree/master/proto-google-cloud-bigtable-admin-v2/src/main/proto/google/bigtable/admin/v2
```

Comment:
should these link to code in `python-bigtable`? For example, https://github.com/googleapis/python-bigtable/blob/master/google/cloud/bigtable_admin_v2/proto/bigtable_table_admin.proto?

Reply:
@kolea2 Yes, I have also doubt about the links but when i have checked links which is in `data-api.rst` file they have redirected to the `java-hbase` and the older link also i have found related of google-jcloud-ava. If you are sure about those links then i will change all the links which redirect to the python relatable content.

Reply:
+1. Should be pointing to protos in the python library.